project-base/storefront/components/Layout/Header/AutocompleteSearch/AutocompleteSearchArticlesResult.tsx

```diff
@@ -0,0 +1,54 @@
+import { SearchResultLink, SearchResultSectionGroup, SearchResultSectionTitle } from './AutocompleteSearchPopup';
+import { AUTOCOMPLETE_ARTICLE_LIMIT } from './constants';
+import { TypeAutocompleteSearchQuery } from 'graphql/requests/search/queries/AutocompleteSearchQuery.generated';
+import { GtmSectionType } from 'gtm/enums/GtmSectionType';
+import { onGtmAutocompleteResultClickEventHandler } from 'gtm/handlers/onGtmAutocompleteResultClickEventHandler';
+import useTranslation from 'next-translate/useTranslation';
+
+type AutocompleteSearchArticlesResultProps = {
+    articlesSearch: TypeAutocompleteSearchQuery['articlesSearch'];
+    onClosePopupCallback: () => void;
+    autocompleteSearchQueryValue: string;
+};
+
+export const AutocompleteSearchArticlesResult: FC<AutocompleteSearchArticlesResultProps> = ({
+    autocompleteSearchQueryValue,
+    articlesSearch,
+    onClosePopupCallback,
+}) => {
+    const { t } = useTranslation();
+
+    if (!articlesSearch.length) {
+        return null;
+    }
+
+    return (
+        <div>
+            <SearchResultSectionTitle>
+                {`${t('Articles')}`}
```

Comment:
I do not see any benefit of using backticks here 🤔

Reply:
Fixed